assign-doi-datacite/src/main/java/no/unit/nva/doi/datacite/clients/DataCiteClient.java

```diff
@@ -172,7 +172,7 @@ private Doi extractDoiPrefixAndSuffix(String createMetadataResponseBody) {
         var identifier = StringUtils.substringBetween(createMetadataResponseBody,
             CHARACTER_PARENTHESES_START,
             CHARACTER_PARENTHESES_STOP);
-        return Doi.builder().identifier(identifier).build();
+        return Doi.builder().withIdentifier(identifier).build();
```

Comment:
TODO: Add proxy as a configuration value. MDS API doesn't answer with URIs. For `test` the proxy host need to go to the handle staging server.

Reply:
@axthosarouris changing the createDoi (draft doi) in client to use the REST API. According to https://support.datacite.org/docs/api-create-dois it should have the DOI URI in `data.attributes.identifiers.identifier`, but this seems to be `[]` in test API endpoint. 

So we still need an environment variable with PROXY set pr. environment matching the API ENDPOINT configuration for the impl. clients.